web/source/kmwdom.ts

```diff
@@ -378,7 +378,7 @@ namespace com.keyman {
 
       if(this.isKMWInput(Pelem)) {
         if(!this.isKMWDisabled(Pelem)) {
-          if(touchable) {
+          if(touchable && !this.keyman.isEmbedded) {
```

Comment:
@jahorton In the Android app, the "enter" button on the OSK seems disabled. Should this be `&& this.keyman.isEmbedded`?

It's affecting all the keyboards I've tried:
SIL EuroLatin
English
Khmer Angkor

Reply:
That shouldn't be the spot responsible - it should definitely *not* be what you suggest, as that breaks other things.  If changing it this way is making a difference, it must be bypassing some other bug somehow.

Reply:
Noting you fixed the "enter" button in 194d105ed23c7f7e57e65403b668a512154216eb